src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <div class="card__main-picture-wrapper">
+        <img
+          class="card__main-picture"
+          src="./images/imac.jpeg"
+          alt="Product picture 'Monitor APPLE A1419 iMac 27'"
+        >
+      </div>
+
+      <h3 class="card__main-title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+
+      <p class="card__item-number">Код товара: 195434</p>
+
+      <div class="stars">
+        <ul class="stars__list stars__list--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <p class="stars__amount">Отзывов: 5</p>
```

Comment:
If we dig in that, that's shouldn't be really a part of `stars` block? I think that reviews count should be connected to `card` block, IMHO

Reply:
Ok, I change this